pom.xml

```diff
@@ -54,5 +54,20 @@
 
     <properties>
         <openmrsPlatformVersion>2.6.0</openmrsPlatformVersion>
+		<webservices.restVersion>2.44.0</webservices.restVersion>
     </properties>
```

Comment:
It looks like the indentation here mixes spaces and tabs. Could we stick with one or the other?